### DISCUSSION TITLE
`--apply` versus `--write`

### DISCUSSION THREAD

**Discussion by rozalily** [biomejs/biome#2323] [Ideas]
`biome check`/`biome lint` use `--apply` to mean that fixes should be applied to files, but `biome format` uses `--write` to mean the same thing (which I assume was done to allow dropping Biome directly into scripts that once used Prettier). They should be standardized in some way. This should probably be done by allowing `--apply` in `biome format`, since `--write` is ambiguous for lint/check due to their concept of safe vs. unsafe.

**Comment by ematipico** [comment]
We have already an issue to track the change https://github.com/biomejs/biome/issues/2267
